mcs/class/corlib/corlib.dll.sources

```diff
@@ -91,8 +91,8 @@ System/AppDomainSetup.cs
 System/ApplicationIdentity.cs
 System/ArgIterator.cs
 System/Array.cs
-System/AssemblyLoadEventArgs.cs
-System/AssemblyLoadEventHandler.cs
+../../../external/corefx/src/Common/src/CoreLib/System/AssemblyLoadEventArgs.cs
```

Comment:
Keeping these `../../../external/corefx` paths mixed with the "local" and the referencesource one looks messy when you look at the `.dll.sources` file. It's already been fixed previously https://github.com/mono/mono/pull/8389. I don't want you to require to change it since it's not really breaking anything, but it would be better over the long run to move these files into their own block with the rest of the `../../../external/corefx` paths.

Reply:
@luhenry ok, I'll re-order them before merge, I am replacing sources via tool I mentioned in the description